migrateBuckets/migrateBuckets.js

```diff
@@ -15,89 +14,90 @@ const logger = new werelogs.Logger('s3utils:migrateBuckets')
     .newRequestLogger();
 const { BucketInfo } = models;
 const versionIdUtils = versioning.VersionID;
-/* expected input JSON structure:
-* {
-*   bucketd: {
-*       bootstrap: <>,
-*   },
-*   https: {
-*       key: <>,
-*       cert: <>,
-*       ca: <>,
-*   }
-* }
-*/
 
 // configurable params
 const ACCESS_KEY = process.env.ACCESS_KEY;
 const SECRET_KEY = process.env.SECRET_KEY;
-const ENDPOINT = process.env.ENDPOINT;
 
-commander
-.version('0.0.1')
+commander.version('1.0.0')
 .option('-b, --bucket-name <bucketName>', 'Name of the bucket')
-.option('-s, --src-canonical-id <srcCanonicalId',
+.option('-s, --src-canonical-id <srcCanonicalId>',
     'Canonical id of source account')
-.option('-d, --dest-canonical-id <destCanonicalId',
+.option('-d, --dest-canonical-id <destCanonicalId>',
     'Canonical id of destination account')
 .option('-o, --dest-owner-name <destOwnerName>',
     'Owner display name of destination account')
-.option('-c, --config-path <configPath>', 'Path to metadata config')
+.option('-e, --endpoint <endpoint>', 'S3Connector endpoint')
+.option('-u, --bucketd <bucketd>', 'Bootstrap list of bucketd endpoints')
+.option('-k, --https-key [httpsKey]', 'HTTPS key')
```

Comment:
minor point: for the http-xxx options I would not have provided a short-option form (single letter) but that's ok if it's there.